src/cq_cam/tests/test_op3d.py

```diff
@@ -12,40 +12,48 @@ def test_profile_square_outside(job: Job, box):
         "G90 G54 G64 G50 G17 G94\nG49 G40 G80\nG21\nG30\n"
         "M3\n"
         "(Job - Surface 3D)\n"
-        "G0Z10\n"
-        "X-1.5Y1\n"
-        "Z1\n"
-        "G1Z0\n"
-        "X-1\n"
-        "X-0.5\n"
-        "X0\n"
-        "X0.5\n"
-        "X1\n"
-        "X1.5\n"
-        "Y0.5\n"
-        "Y0.2\n"
-        "X1\n"
-        "X0.5\n"
-        "X0\n"
-        "X-0.5\n"
-        "X-1\n"
-        "X-1.5\n"
-        "Y-0.3\n"
-        "Y-0.6\n"
-        "X-1\n"
-        "X-0.5\n"
-        "X0\n"
-        "X0.5\n"
-        "X1\n"
-        "X1.5\n"
-        "Y-1.1\n"
-        "Y-1.4\n"
-        "X1\n"
-        "X0.5\n"
-        "X0\n"
-        "X-0.5\n"
-        "X-1\n"
-        "X-1.5\n"
+        "G0 Z10\n"
+        "G0 X-1.5 Y1\n"
+        "G0 Z1\n"
+        "G1 Z0\n"
+        "G1 X-1.5 Y1 Z0\n"
```

Comment:
`G1 X-1.5 Y1 Z0` is redundant. Feature or bug?

Reply:
It is a bug that was present in profile and pocket as well and has been fixed since. But since Drill and Surface3D and not properly integrated with the fluent API I need to change it for them as well. Should I or should we do that when we refactor them? @voneiden